package.json

```diff
@@ -74,6 +74,7 @@
         "moment": "^2.29.4",
         "nouislider": "11.0.3",
         "numeral": "^2.0.6",
+        "primeicons": "^7.0.0",
```

Comment:
A little silly, but probably want to keep the 8.0.0.md added dependencies up to date

Reply:
oh yeah, ++